CHANGELOG.md

```diff
@@ -2,12 +2,42 @@
 
 ## Unreleased
 
+## 3.13.0 (Feature release)
+
+### New features
+
+#### Add a `nonce` attribute to inline scripts
+
+If your service has a Content Security Policy (CSP), you can now add a `nonce` attribute to inline scripts within the Nunjucks page template. To do this, set the `cspNonce` Nunjucks variable. However, you should only add this attribute if you're not able to [include the hash for the inline scripts in your CSP](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#if-your-javascript-is-not-working-properly).
+
+Make sure you [understand the security implications of using this attribute](https://www.w3.org/TR/CSP/#security-considerations), as wrong implementation could affect your service’s security.
+
+[Read more about how to configure your CSP to work with GOV.UK Frontend](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#select-and-initialise-part-of-a-page).
+
+This was added in [pull request #2245: Allow the CSP `nonce` attribute to be set on the inline script in the page template](https://github.com/alphagov/govuk-frontend/pull/2245). Thanks to [@natcarey](https://github.com/natcarey) for contributing this issue and its solution.
+
+#### Add a 'none' option and 'or' divider to checkboxes
+
+You can now add a 'none' option to checkboxes. For example, 'None of the above' or 'None of these options apply to me.'
+
+Use the 'none' option to [allow users to state that none of the checkboxes apply to them](https://design-system.service.gov.uk/components/checkboxes/#add-an-option-for-none). Otherwise, users would have to work out that they should leave all the checkboxes unticked.
```

Comment:
Basing the link on the subheading in line 68 of the [pull request for the content](https://github.com/alphagov/govuk-design-system/pull/1535/files).

Reply:
Unfortunately I think there's a bug in the code that turns the headings into anchor links. Because of the quote after 'none' (I assume) the actual anchor link has a stray `-` at the end which will need including here too.

```suggestion
Use the 'none' option to [allow users to state that none of the checkboxes apply to them](https://design-system.service.gov.uk/components/checkboxes/#add-an-option-for-none-). Otherwise, users would have to work out that they should leave all the checkboxes unticked.
```